rfcs/0005-pablo-distribution-new.md

```diff
@@ -0,0 +1,178 @@
+# Design Proposal: Pablo Fees & Staking Rewards Distribution
```

Comment:
Nice, though I think it's good to migrate this over as a separate PR/task as currently a major bugfix is held up with this PR :)

Reply:
These changes weren't intended to be committed, I meant to stash them. They can be removed 👍

Reply:
I will separate them out, thanks